client-src/elements/form-field-specs.js

```diff
@@ -1134,6 +1141,51 @@ export const ALL_FIELDS = {
     help_text: '',
   },
 
+  'enterprise_policies': {
+    type: 'input',
+    attrs: MULTI_STRING_FIELD_ATTRS,
+    required: false,
+    label: 'Enterprise policies',
+    help_text: html`
+      List of policies that control the feature, if any.`,
```

Comment:
What is the syntax of each item in the list?  Can we give an example?

Reply:
MULTI_STRING_FIELD_ATTRS I put an example in the placeholder. But it is a list comma separated of pascal case policy names.
Would the placeholder example be enough ? Or should I repeat it in the help text ?

Reply:
OK, I saw that but I didn't know if that was an example of the syntax expected or more of a meta-syntactic variable.   If the feature owners will recognize it, that is fine.